chat-plugins/poll.js

```diff
@@ -5,7 +5,7 @@
  */
 
 'use strict';
-
+let activePolls = 0;
```

Comment:
imo scrap the activePolls stuff (ill say how to fix the deletion correctly in a bit)